### PR TITLE
[21760] Remove modules and custom fields from project settings overview

### PR DESCRIPTION
**Pull request by leewaa**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://leewaa-21760-remove-modules-and-customfields.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://leewaa-21760-remove-modules-and-customfields.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by leewaa** [comment]
This is for [21760](https://community.openproject.org/work_packages/21760/) but also fixes [21755
](https://community.openproject.org/work_packages/21755/)

Once green please review/merge

**Comment by leewaa** [comment]
@why-el review/merge when you're ready :)

File: app/views/projects/new.html.erb [hunk 1]
```diff
@@ -29,10 +29,13 @@ See doc/COPYRIGHT.rdoc for more details.
 <% html_title l("label_project_new") %>
 <%= toolbar title: l(:label_project_new) %>
 <%= labelled_tabular_form_for @project do |f| %>
-  <%= render :partial => 'form', :locals => { :f => f,
-                                              :project => @project,
-                                              :renderTypes => true,
-                                              :show_all_fields => false } %>
+  <%= render partial: 'form', :locals: { f: f,
```
Comment (oliverguenther) [inline app/views/projects/new.html.erb:32]: `:locals:` ? This breaks current dev specs ;) I've removed this in https://github.com/opf/openproject/commit/0bb88ddb2a5dd33e054ff99dfaba595606aade2b

**Pull request merged**